assets/js/modules/analytics/datastore/profiles.js

```diff
@@ -241,19 +274,19 @@ export const reducer = ( state, { type, payload } ) => {
 			};
 		}
 
-		case RECEIVE_PROFILES: {
-			const { profiles } = payload;
+		case START_FETCH_PROFILES: {
+			const { accountID, propertyID } = payload;
 
 			return {
 				...state,
-				profiles: {
-					...state.profiles,
-					...groupBy( profiles, ( { accountId, webPropertyId } ) => `${ accountId }::${ webPropertyId }` ), // Capitalization rule exception: `accountId` and `webPropertyId` are properties of an API returned value.
+				isFetchingProfiles: {
+					...state.isFetchingProfiles,
+					[ `${ accountID }::${ propertyID }` ]: true,
```

Comment:
Don't we have a utility function to handle this sort of "key serialisation" now? I couldn't find it, but I could've sworn @felixarntz introduced one recently.

Reply:
We do (`stringifyObject`), however I think using strings like the one here is actually preferable for simple cases like this. While `stringifyObject` _could_ be used here, it generates an md5-based value which is much harder to read, and furthermore it only works on an object, so we'd need to turn these values into an object. I think for simple individual arguments doing the manual concatenation like here makes sense.